internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -183,6 +179,7 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 		}, w.Blacklist)
 	}
 
+	// todo: remove when a upper version be bumped
```

Comment:
it might be a good idea to file an issue for this and put a "backwards incompatible" label, or put this in the 2.0 milestone. Otherwise, we will probably forget this when we bump to 2.0.0

Reply:
Done : https://github.com/operator-framework/operator-sdk/issues/4330